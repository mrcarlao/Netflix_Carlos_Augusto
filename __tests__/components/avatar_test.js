import React from "react";
import renderer from "react-test-renderer";
import Avatar from "../../components/Avatar";


test("Criando Component", () => {
    expect(renderer.create(<Avatar />)).toMatchSnapshot();

});