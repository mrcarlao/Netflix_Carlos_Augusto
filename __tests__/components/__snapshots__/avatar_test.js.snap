// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Criando Component 1`] = `
<View
  accessible={true}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "column",
      "opacity": 1,
    }
  }
>
  <View>
    <Image
      style={
        Array [
          Object {
            "height": 60,
            "marginBottom": 10,
            "marginLeft": 10,
            "marginRight": 10,
            "marginTop": 10,
            "width": 60,
          },
        ]
      }
    />
  </View>
  <Text
    style={
      Array [
        Object {
          "color": "white",
        },
      ]
    }
  />
</View>
`;
